gef-ui/server/controllers/application-details/index.js

```diff
@@ -24,7 +24,7 @@ const getUserAuthorisationLevelsToApplication = require('../../utils/user-author
 const { FACILITY_TYPE, AUTHORISATION_LEVEL, DEAL_SUBMISSION_TYPE, STAGE } = require('../../constants');
 const Application = require('../../models/application');
 const { MAKER } = require('../../constants/roles');
-const { canUserAmendIssuedFacilities } = require('../../utils/canUserAmendIssuedFacilities');
+const { canUserAmendIssuedFacilities } = require('../../utils/facility-amendments.helper');
```

Comment:
Is there a unit test for this? Would be surprised if the test is passing with this change

Reply:
The unit tests exist but they use `Object.containing` so are still passing without checking for `canUserAmendIssuedFacilities`

Reply:
i remember, thanks